admin/class-admin-apple-sections.php

```diff
@@ -181,7 +181,7 @@ public static function get_sections_for_post( $post_id, $format = 'url' ) {
 			$post_sections[] = reset( $sections );
 		}
 
-		return $post_sections;
+		return apply_filters( 'get_sections_for_post', $post_sections, $post_id, $format );
```

Comment:
```suggestion
		/**
		 * Filters the sections for a post.
		 *
		 * @since 2.1.0
		 *
		 * @param array  $post_sections The sections for the post.
		 * @param int    $post_id       The post ID.
		 * @param string $format        The section format (e.g., 'url').
		 */
		return apply_filters( 'get_sections_for_post', $post_sections, $post_id, $format );
```